build-tools/plots-to-appinsights/App.config

```diff
@@ -0,0 +1,14 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Can kill this file, it doesn't do anything.

Reply:
Indentation is also wonky, but killing the file will fix the indentation. :-)

Reply:
So long App.config. Commit: 71c3a38587a0dc8e4cd76b74b3302acb980484bf